README.md

```diff
@@ -25,6 +25,19 @@ Both sites are powered by same underlying monitoring infrastructure. At the mome
 
 Thanks for being interested! First, check out the [issues list](https://github.com/codeforamerica/snap-it-up/issues). Some of the work there is discussion, exploration, and thinking. Feel free to pose questions or read through the discussions if you'd like to pitch in or learn more. Some of the issues are about the actual source code in this repo. If you'd like to contribute code or suggest new things to be added to the site, feel free to add to those issues.
 
+## Local Development
+
+Follow these steps in order to run the application locally:
+
+0. Have a Ruby development environment created. These steps assume you are using Bundler for package management and Foreman managing environment variables.
+1. Copy `.env-sample` to `.env`. This is where you will be saving configuration for steps 2 and 3.
+2. Provision a Mongo database. This can either be done locally, or by setting up a free cloud database through [MongoLab](https://mongolab.com)
+3. Create a [Pingometer](http://pingometer.com) account. 
+4. Create a new pingometer check for a website listed in `public/data/pingometer_monitors.json` with 1 minute resolution and make note of its id (look in the url: `monitors/{monitor_id}`)
+5. Update `public/data/pingometer_monitors.json` with your new check's id (be sure NOT to ever git commit this change)
+6. Run `$ foreman run bundle exec rake pingometer:load_events[{monitor_id}]`. If you set up the previous correctly, it should run successfully
```

Comment:
And, if it would helpful, I’m totally open to renaming those tasks, maybe putting them together in one namespace, or creating one task that just calls those. They might make more sense unified around a them of “updating from external data sources” than “pingometer stuff, s3 stuff, and analysis stuff.”